src/server/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
Likely not the approach we want, I just "made it work". If we like the Jest change, I'd love input on how to handle this stuff. Didn't see any docs/explanations anywhere, but might just be my search skills.

Reply:
Pushed a better approach